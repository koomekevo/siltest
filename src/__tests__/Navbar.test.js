/* eslint-disable */
import TestRenderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter as Router } from "react-router-dom";
import Navbar from "../components/Navbar";
import React from "react";

describe("Renders the Navbar", () => {
  it("Renders the Navbar", () => {
    const navbar = TestRenderer.create(
      <Router>
        <Navbar />
      </Router>
    ).toJSON();
    expect(navbar).toMatchSnapshot();
  });
});
