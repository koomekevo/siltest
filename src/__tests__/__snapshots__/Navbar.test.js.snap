// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Navbar Renders the Navbar 1`] = `
<nav
  className="bg-white opacity-80 border-black border-b-2 px-2 sm:px-4 py-2.5 dark:bg-gray-900"
>
  <div
    className="container flex flex-wrap items-center justify-between mx-auto"
  >
    <a
      className="flex items-center"
      href="/"
    >
      <img
        alt="SILTest Logo"
        className="h-6 mr-3 sm:h-9"
        src="logo.png"
      />
      <span
        className="self-center text-xl font-semibold whitespace-nowrap dark:text-white"
      >
        SILTest
      </span>
    </a>
    <div
      className="flex md:order-2"
    >
      <button
        className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center mr-3 md:mr-0 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
        type="button"
      >
        <a
          className=""
          exact={true}
          href="/SignIn"
          onClick={[Function]}
        >
          SIGN IN
        </a>
      </button>
      <button
        aria-controls="navbar-sticky"
        aria-expanded="false"
        className="inline-flex items-center p-2 text-sm text-gray-500 rounded-lg md:hidden hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-600"
        data-collapse-toggle="navbar-sticky"
        type="button"
      >
        <span
          className="sr-only"
        >
          Open main menu
        </span>
        <svg
          aria-hidden="true"
          className="w-6 h-6"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </div>
    <div
      className="items-center justify-between hidden w-full md:flex md:w-auto md:order-1"
      id="navbar-sticky"
    >
      <ul
        className="flex flex-col p-4 mt-4 border border-gray-100 rounded-lg bg-gray-50 md:flex-row md:space-x-8 md:mt-0 md:text-sm md:font-medium md:border-0 md:bg-white dark:bg-gray-800 md:dark:bg-gray-900 dark:border-gray-700"
      >
        <li>
          <a
            aria-current="page"
            className="block py-2 pl-3 pr-4 text-white bg-blue-700 rounded md:bg-transparent md:text-blue-700 md:p-0 dark:text-white active"
            exact={true}
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            className="block py-2 pl-3 pr-4 text-gray-700 rounded hover:bg-gray-100 md:hover:bg-transparent md:hover:text-blue-700 md:p-0 md:dark:hover:text-white dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:dark:hover:bg-transparent dark:border-gray-700"
            exact={true}
            href="/About"
            onClick={[Function]}
          >
            About
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
